packages/react-components/src/components/SendBox.tsx

```diff
@@ -242,9 +248,26 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     // Message can be empty if there is a valid attachment upload
     if (
       sanitizeText(message).length > 0 ||
-      /* @conditional-compile-remove(attachment-upload) */ hasCompletedAttachmentUploads(attachmentsWithProgress)
+      /* @conditional-compile-remove(attachment-upload) */ isAttachmentUploadCompleted(attachmentsWithProgress)
     ) {
-      onSendMessage && onSendMessage(message);
+      onSendMessage &&
+        onSendMessage(
+          message,
+          /* @conditional-compile-remove(attachment-upload) */
+          {
+            attachmentMetadata: attachmentsWithProgress
+              ?.filter((v) => {
+                return !('error' in v) && !v.error?.message;
```

Comment:
I'm wondering if this check is corrent:
`!('error' in v)` already checks that there is no `error` property in `v`, so we will never need to check `!v.error?.message`

Reply:
the first condition check if the key error exists, the second condition checks if the key contains valid value, without second condition, this will be valid but it shouldn't: 
```
{ error: {} }
```